### PR TITLE
chore: Added unittest to check for specific repo files

### LINKED ISSUE #39: Add unittest to check for specific repo files
This repo should have the following list of files included:
- ./Docker or docker/Docker
- ./docker-compose.yml or ./docker/docker-compose.yml
- ./.env_sample
- ./.gitignore
- ./.travis.yml
- ./.codeclimate.yml
- ./CHANGELOG.md
- ./CODE_OF_CONDUCT.md
- ./CONTRIBUTING.md
- ./.github/ISSUE_TEMPLATE
- ./LICENSE.md
- ./.github/PULL_REQUEST_TEMPLATE
- ./README.md
- ./TROUBLESHOOTING.md
- ./USAGE.md
- ./USE_CASES.md

This PR is only asking for tests of the existence of these files, if the files do not exist when you run the tests - do not worry about the tests not passing. We will identify this and create a new PR for the issue.
Thank you!

### PR DESCRIPTION
**Pull request by mptap**
# Fixes #39 

### Checklist
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guide] and my PR follows them.
- [x] I updated my branch with the master branch.
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation about the functionality in the appropriate .md file
- [x] I have added in line documentation to the code I modified

### Short description of what this PR does:
- Added unittest to check for specific repo files mentioned in #39 

If you have questions, please send an email to [Sendgrid](mailto:dx@sendgrid.com), or file a Github Issue in this repository.

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/smtpapi-ruby?pullRequest=42) <br/>All committers have signed the CLA.

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/smtpapi-ruby?pullRequest=42) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/smtpapi-ruby?pullRequest=42) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/smtpapi-ruby?pullRequest=42) it.</sub>

File: test/test.rb [hunk 1]
```diff
@@ -202,6 +202,69 @@ def test_ip_pool
...
+    assert(File.file?('./TROUBLESHOOTING.md'))
+  end
+
+  def test_usage_exists
+    assert(File.file?('./USAGE.md'))
+  end
+
+  def test_use_cases_exists
+    assert(File.file?('./USE_CASES.md'))
+
```
Comment (mbernier) [inline test/test.rb]: Looks like we might be missing an `end` here, based on this error:
```
$ rake test
/home/travis/.rvm/rubies/ruby-2.2.7/bin/ruby -w -I"lib:test" -I"/home/travis/.rvm/gems/ruby-2.2.7/gems/rake-12.2.1/lib" "/home/travis/.rvm/gems/ruby-2.2.7/gems/rake-12.2.1/lib/rake/rake_test_loader.rb" "test/test.rb" 
/home/travis/build/sendgrid/smtpapi-ruby/test/test.rb:274: warning: mismatched indentations at 'end' with 'def' at 265
/home/travis/.rvm/rubies/ruby-2.2.7/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:55:in `require': /home/travis/build/sendgrid/smtpapi-ruby/test/test.rb:274: syntax error, unexpected end-of-input, expecting keyword_end (SyntaxError)
	from /home/travis/.rvm/rubies/ruby-2.2.7/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /home/travis/.rvm/gems/ruby-2.2.7/gems/rake-12.2.1/lib/rake/rake_test_loader.rb:17:in `block in <main>'
	from /home/travis/.rvm/gems/ruby-2.2.7/gems/rake-12.2.1/lib/rake/rake_test_loader.rb:5:in `select'
	from /home/travis/.rvm/gems/ruby-2.2.7/gems/rake-12.2.1/lib/rake/rake_test_loader.rb:5:in `<main>'
rake aborted!
Command failed with status (1): [ruby -w -I"lib:test" -I"/home/travis/.rvm/gems/ruby-2.2.7/gems/rake-12.2.1/lib" "/home/travis/.rvm/gems/ruby-2.2.7/gems/rake-12.2.1/lib/rake/rake_test_loader.rb" "test/test.rb" ]
/home/travis/.rvm/gems/ruby-2.2.7/gems/rake-12.2.1/exe/rake:27:in `<top (required)>'
/home/travis/.rvm/gems/ruby-2.2.7/bin/ruby_executable_hooks:15:in `eval'
/home/travis/.rvm/gems/ruby-2.2.7/bin/ruby_executable_hooks:15:in `<main>'
Tasks: TOP => test
(See full trace by running task with --trace)
```

**Review by mbernier — CHANGES_REQUESTED** [review]

**Review by mbernier — APPROVED** [review]

**Comment by thinkingserious** [comment]
Hello @mptap,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Comment by mbernier** [comment]
@mptap 

If you are interested, we would love the opportunity to talk to you about Hacktoberfest and our API libraries.

Our agenda would be: Explore what you liked and is there anything we can do to improve? 

You can grab a time [on my calendar](https://calendly.com/matt-bernier/30min) that works for you and we can have a chat on Google Hangout or Skype. If you prefer, you can email me using my GitHub username at my company’s domain.

Thank you so much,

Matt Bernier - @mbernier - SendGrid Developer Experience Product Manager
Elmer Thomas - @thinkingserious - SendGrid Developer Experience Engineer
@SendGrid

**Comment by mptap** [comment]
Done @mbernier, have put a time on your calendar for Mon Nov 20th at
12:30pm PST.

Thank you,

Manjiri Tapaswi
Graduate Student | Computer Science | NCSU

**Pull request merged**